models/adlink_test.go

```diff
@@ -0,0 +1,156 @@
+package models_test
+
+import (
+	"go-google-scraper-challenge/initializers"
+	"go-google-scraper-challenge/models"
+	. "go-google-scraper-challenge/tests/helpers"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("AdLink", func() {
+	Describe("CreateAdLink", func() {
```

Comment:
```suggestion
	Describe("#CreateAdLink", func() {
```

Reply:
@junan Nice catch 🙌🏼 , fixed in [40e378e](https://github.com/carryall/go-google-scraper-challenge/pull/42/commits/40e378e86ec5049b1a9b7ca8b48715c9fd2ac087)